From ce959542299efffd672bfade91677628dbd7869d Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Mon, 16 Apr 2018 15:25:56 +0300
Subject: [PATCH 5/6] Remove adb.secure recovery property context

* This is a non-existent property, recovery
  was trying to set ro.adb.secure instead,
  which is forbidden

Change-Id: I034dd16021f3f0244a1b1cd6e12ec32bcf73a9a7
---
 common/private/property_contexts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/common/private/property_contexts b/common/private/property_contexts
index cbfadff..4ad07d6 100644
--- a/common/private/property_contexts
+++ b/common/private/property_contexts
@@ -1,4 +1,3 @@
 adb.network.port              u:object_r:adbtcp_prop:s0
-adb.secure                    u:object_r:lineage_recovery_prop:s0
 lineage.userinit.active       u:object_r:userinit_prop:s0
 recovery.perf.mode            u:object_r:lineage_recovery_prop:s0
-- 
2.15.1

